css-canvas-medical/styles.css

```diff
@@ -0,0 +1,206 @@
+.container {
+  width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  padding-left: 60px;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.foot-buttons {
+  text-decoration-line: none;
+  margin-right: 25px;
+}
+
+#footer {
+  background-color: #fafafa;
+  padding-top: 35px;
+  padding-bottom: 35px;
+  padding-left: 60px;
+}
+
+#get-started {
+  padding: 18px 45px;
+  background-color: #294464;
+  color: white;
+  text-decoration-line: none;
+  border-radius: 4px;
+  float: right;
+  margin-top: 25px;
+}
+
+h3,
+p,
+.foot-buttons {
+  color: #6f6f6f;
+}
+
+#ready-row {
+  padding-top: 20px;
+  padding-bottom: 20px;
+  line-height: 0.5;
+}
+
+h3 {
+  font-weight: lighter;
+  font-size: 28px;
+}
+
+#clinicians-row {
+  flex-direction: column;
+  text-align: center;
+  background-color: #fafafa;
+}
+
+.margin-bottom {
+  margin-bottom: 0;
+}
+
+.whole-row {
+  background-color: #fafafa;
+  padding-top: 25px;
+}
+
+.location {
+  text-align: center;
+  margin-top: 25px;
+}
+
+.location-row {
+  padding-bottom: 70px;
+  padding-top: 45px;
+}
+
+#clinician {
+  width: 400px;
+  margin: auto;
+  margin-top: 15px;
+}
+
+.column-one-fourth {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 25%;
+}
+
+.address-row {
+  background: #525351;
+  color: white;
+  padding-top: 60px;
+  padding-bottom: 100px;
+  font-family: 'Avenir', sans-serif;
+}
+
+.company-button,
+.product-button {
+  color: white;
+  text-decoration-line: none;
+  line-height: 35px;
+}
+
+.address-line {
+  margin: 0;
+  color: white;
+  font-size: 14px;
+}
+
+h4 {
+  text-transform: uppercase;
+  margin-bottom: 10px;
+}
+
+#canvas-bottom {
+  width: 115px;
+  margin-top: 22px;
+  margin-bottom: 45px;
+}
+
+.nav-button {
+  text-decoration-line: none;
+  color: #c0c0c0;
+  font-family: 'Avenir', sans-serif;
+  margin-left: 28px;
+}
+
+#contact-us {
+  background-color: #294464;
+  color: white;
+  border-radius: 4px;
+  padding: 10px;
+}
+
+#top-right-bar {
+  display: flex;
+  justify-self: center;
+  font-size: 14px;
+  align-items: center;
+}
+
+.top-row {
+  border-style: solid;
+  border-color: #eaecef;
+  border-left: 0;
+  border-right: 0;
+  border-top: 0;
+  border-width: thin;
+  padding-bottom: 10px;
+  margin-top: 15px;
+}
+
+#canvas-top {
+  width: 115px;
+}
+
+#top-left-bar {
+  display: flex;
+}
+
+.hero {
+  width: 94%;
+  position: absolute;
+  opacity: 0.85;
+  height: 470px;
+}
+
+.hero-row {
+  display: flex;
+  position: relative;
+  margin-bottom: 530px;
```

Comment:
Woah, that's a big one!! Why is this necessary? We may be able to solve this a different way